frontend/integration-tests/tests/kubevirt/clone.vm.scenario.ts

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable no-undef, max-nested-callbacks */
+import { execSync } from 'child_process';
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+
+// eslint-disable-next-line no-unused-vars
+import { removeLeakedResources, waitForCount, searchYAML, VM_BOOTUP_TIMEOUT, PAGE_LOAD_TIMEOUT } from './utils';
+import { appHost, testName } from '../../protractor.conf';
+import { filterForName, isLoaded, resourceRowsPresent, resourceRows } from '../../views/crud.view';
+import { basicVmConfig, networkInterface, testNad, getVmManifest, hddDisk, cloudInitCustomScriptConfig } from './mocks';
+import * as wizardView from '../../views/kubevirt/wizard.view';
+import Wizard from './models/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+import { nicsTab, disksTab } from '../../views/kubevirt/virtualMachine.view';
+
+
+describe('Test clone VM.', () => {
+  const wizard = new Wizard();
+  const testContainerVm = getVmManifest('Container');
+  const vm = new VirtualMachine(testContainerVm.metadata.name, testContainerVm.metadata.namespace);
+
+  describe('Test Clone VM wizard dialog.', () => {
+    beforeAll(async() => {
+      execSync(`echo '${JSON.stringify(testContainerVm)}' | kubectl create -f -`);
+      execSync(`echo '${JSON.stringify(testNad)}' | kubectl create -f -`);
+    });
+
+    afterAll(async() => {
+      execSync(`kubectl delete -n ${testName} net-attach-def ${testNad.metadata.name}`);
+      execSync(`kubectl delete -n ${testName} vm ${testContainerVm.metadata.name}`);
+    });
+
+    it('Display warning in clone wizard when cloned vm is running.', async() => {
```

Comment:
can you also check that Clone button is still enabled and when clicked, VM is stopped and cloned ?

Reply:
Done.